website/docs/r/api_gateway_resource.html.markdown

```diff
@@ -37,5 +37,7 @@ The following arguments are supported:
 
 The following attributes are exported:
 
-* `id` - The ID of the API resource
-* `path` - The complete path for this API resource, including all parent paths
+* `id` - The resource's identifier.
+* `parent_id` - The parent resource's identifier.
+* `path` - The complete path for this API resource, including all parent paths.
+* `path_parth` - The last path segment for this resource.
```

Comment:
This does no harm, but we tend to avoid documenting attributes which are exposed because they're also arguments. i.e. we don't double-document the same fields as arguments and attributes.

Instead we could just use different wording for the sentence above:

> In addition to all arguments above, the following attributes are exported: